lib/index.js

```diff
@@ -314,12 +314,22 @@ class GalleryImageArticle extends BaseAsset {
         }
     }
 
+    set_content_dates(value) {
+        this._content_dates = value.map((v) => {
+            if (!(v instanceof Date)) {
+                return new Date(v);
+            }
+
+            return v;
+        });
+    }
+
     to_metadata() {
         const metadata = super.to_metadata();
         Object.assign(metadata, {
             'document': this._document.html(),
             'authors': [this._author],
-            'published': this._date_published.toISOString(),
```

Comment:
Is it possible to make that null?

The reason I ask is that it looks like on the website there is no date indicating when the article itself was published. I wouldn't want to use the content date, since otherwise it would get used for sorting.